python/ray/llm/_internal/serve/deployments/prefill_decode_disagg/prefill_decode_disagg.py

```diff
@@ -160,13 +151,6 @@ async def _predict(
         ):
             yield chunk
 
-    async def check_health(self) -> None:
```

Comment:
These must be removed. In general the health check of a deployment is not bounded to the health check of its child deployments.

Reply:
cc @lk-chen fyi